azure-devops/azext_devops/dev/pipelines/pipeline.py

```diff
@@ -123,20 +146,35 @@ def pipeline_run(id=None, branch=None, commit_id=None, name=None, open=False, va
         raise ValueError('Either the --id argument or the --name argument ' +
                          'must be supplied for this command.')
     client = get_build_client(organization)
+
     if id is None:
         id = get_definition_id_from_name(name, client, project, folder_path)
+
+    if parameters:
+        client = get_new_pipeline_client_v60(organization)
```

Comment:
add some more logging which can tell which code path is being taken, if a customer reports with debug log then that will be helpful